fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/domain/SearchQueryRenderer.java

```diff
@@ -2088,4 +2361,28 @@ private void addAggregateAndOrderByExpressions(QueryData queryData, String code,
         logger.exiting(CLASSNAME, METHODNAME);
         return attributeNames;
     }
+
+    /**
+     * Check if whole-system search.
+     * @param resourceType
+     * @return true if whole-system search, false otherwise
+     */
+    private boolean isWholeSystemSearch(String resourceType) {
+        return Resource.class.getSimpleName().equals(resourceType);
+    }
+
+    /**
+     * Check if query parameter is a _tag parameter whose modifier requires
+     * search of the tag's value.
+     * @param queryParm
+     * @return true if tag value search, false otherwise
+     */
+    private boolean isTagValueSearch(QueryParameter queryParm) {
```

Comment:
I don't totally understand this one.  Why is a tag search with no modifier not a "Tag Value Search"?

Reply:
I've removed this method since it only got called in one place after I re-factored the code. What I meant by this is that for the :text, :in, :not-in, :above, and :below modifiers, we need access to the token value. In the new global tables for _tag and _security, we only have a common_token_value_id, so that means we have to join to the common_token_values table to access the token value.